kubernetes-itests/src/test/java/io/fabric8/kubernetes/PodSecurityPolicyIT.java

```diff
@@ -0,0 +1,131 @@
+      .build();
+
+    client.podSecurityPolicies().create(podSecurityPolicy);
+  }
+
+  @Test
+  public void load() {
+
+    PodSecurityPolicy loadedPodSecurityPolicy = client.podSecurityPolicies()
+      .load(getClass().getResourceAsStream("/test-podsecuritypolicy.yml")).get();
+
+    assertNotNull(loadedPodSecurityPolicy);
+    assertEquals("example", loadedPodSecurityPolicy.getMetadata().getName());
+    assertFalse(loadedPodSecurityPolicy.getSpec().getPrivileged());
+    assertEquals("RunAsAny", loadedPodSecurityPolicy.getSpec().getRunAsUser().getRule());
+    assertEquals("RunAsAny", loadedPodSecurityPolicy.getSpec().getFsGroup().getRule());
+    assertEquals("RunAsAny", loadedPodSecurityPolicy.getSpec().getSeLinux().getRule());
+    assertEquals("RunAsAny", loadedPodSecurityPolicy.getSpec().getSupplementalGroups().getRule());
+  }
+
+  @Test
+  public void get() {
+
+    PodSecurityPolicy getPodSecurityPolicy = client.podSecurityPolicies()
+      .withName("test-example").get();
+    assertNotNull(getPodSecurityPolicy);
+    assertEquals("test-example", getPodSecurityPolicy.getMetadata().getName());
+  }
+
+  @Test
+  public void list() {
+
+    PodSecurityPolicyList podSecurityPolicyList = client.podSecurityPolicies().list();
+    assertNotNull(podSecurityPolicyList);
+    assertEquals(1,podSecurityPolicyList.getItems().size());
+    assertEquals("test-example",podSecurityPolicyList.getItems().get(0).getMetadata().getName());
+    assertEquals("RunAsAny",podSecurityPolicyList.getItems().get(0).getSpec().getRunAsUser().getRule());
+    assertEquals("RunAsAny",podSecurityPolicyList.getItems().get(0).getSpec().getFsGroup().getRule());
+    assertEquals("RunAsAny",podSecurityPolicyList.getItems().get(0).getSpec().getSeLinux().getRule());
+    assertEquals("RunAsAny",podSecurityPolicyList.getItems().get(0).getSpec().getSupplementalGroups().getRule());
+  }
+
+  @Test
+  public void update(){
+
+    podSecurityPolicy = client.podSecurityPolicies().withName("test-example").edit()
+      .editSpec().withPrivileged(true).endSpec()
+      .done();
+
+    assertNotNull(podSecurityPolicy);
+    assertEquals("test-example",podSecurityPolicy.getMetadata().getName());
+    assertTrue(podSecurityPolicy.getSpec().getPrivileged());
+    assertEquals("RunAsAny",podSecurityPolicy.getSpec().getRunAsUser().getRule());
+    assertEquals("RunAsAny",podSecurityPolicy.getSpec().getFsGroup().getRule());
+    assertEquals("RunAsAny",podSecurityPolicy.getSpec().getSeLinux().getRule());
+    assertEquals("RunAsAny",podSecurityPolicy.getSpec().getSupplementalGroups().getRule());
+  }
+
+  @Test
+  public void delete(){
+    boolean deleted = client.podSecurityPolicies().delete(podSecurityPolicy);
+    assertTrue(deleted);
+    PodSecurityPolicyList podSecurityPolicyList = client.podSecurityPolicies().list();
+    assertEquals(0,podSecurityPolicyList.getItems().size());
+  }
+
+  @After
+  public void cleanup() {
+    client.podSecurityPolicies().delete();
```

Comment:
I missed this in the initial part of the review. It's not a good idea to delete all the PodSecurityPolicy objects since they are not a namespace specific resource. Instead, you should simply delete the resource you created during test.

Reply:
Yes @rohanKanojia, you seem correct will send a quick PR.